scripts/format_new.py

```
@@ -0,0 +1,170 @@
+    def __init__(self, file_path: Path):
+        self.file_path = file_path
+        self.contacts = self._load_contacts()
+    
+    def _load_contacts(self):
+        """Load contacts from file."""
+        if not self.file_path.exists():
+            logger.warning(f"Contact store file not found: {self.file_path}")
+            return {}
+        
+        try:
+            with open(self.file_path, "r") as f:
```

Ruff (0.14.2)

`open()` should be replaced by `Path.open()`

(PTH123)